docs/jsonrpc_api.md

```diff
@@ -152,3 +153,32 @@ Attempts to look up content key in Trin node's local db.
   }
 }
 ```
+
+# General
+
+## `portal_paginateLocalContentKeys`
+Return a paginated list of all of the content keys (from every subnetwork) currently available in local storage.
+
+### Parameters
+- `offset`: Pagination offset.
+- `limit`: Number of entries to return.
+
+### Returns
+- `content_keys`: List of content keys.
+- `offset`: Pagination offset used.
+- `limit`: Number of entries returned.
+- `total_entries`: Total number of content keys in local storage.
+
+#### Example
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": {
+    "content_keys": ["0x0055b11b918355b1ef9c5db810302ebad0bf2544255b530cdce90674d5887bb286"],
+    "offset": 0,
+    "limit": 5,
```

Comment:
Why are we returning the same offset and limit they just passed in? They should already have it, so it doesn't seem to add any value.

Reply:
This was the recommended scheme in the blog I read about api response pagination. But, I agree it's redundant. Happy to remove it if you don't think it's useful.

Reply:
Went ahead and removed it for now, we can add it back if it ends up being useful